charts/mercure/templates/serviceMonitor.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "mercure.fullname" . }}-metrics
+  labels:
+    app.kubernetes.io/name: {{ include "mercure.name" . }}
+spec:
+  selector:
+    matchLabels:
+      prometheus-operated: "true"
+  endpoints:
```

Comment:
We should likely include options similar to these ones too:  https://github.com/bitnami/charts/blob/1dccc11c8e8cfa80aad6fc6628d1edc02c49b4a4/bitnami/wordpress/templates/servicemonitor.yaml#L22-L36